index.js

```diff
@@ -521,6 +541,12 @@
     return typeof other === typeof this && other.valueOf() === this.valueOf();
   }
 
+  //  Boolean$prototype$lte :: Boolean ~> Boolean -> Boolean
+  function Boolean$prototype$lte(other) {
+    return typeof other === typeof this &&
+      (!this.valueOf() || other.valueOf());
+  }
+
```

Comment:
Ok. Seems arbitrary to care about the difference between Boolean(true) and true - the Boolean algebra we're attempting to express wasn't written with four constructors in mind!